src/rvoice/fluid_rvoice_mixer.c

```diff
@@ -310,6 +310,21 @@ fluid_rvoice_buffers_mix(fluid_rvoice_buffers_t* buffers,
                          int start_block, int sample_count, 
                          fluid_real_t** dest_bufs, int dest_bufcount)
 {
+  /*
+   * SF2.04 9.1.7 Envelope Generators:
+   * 
+   * "When 96 dB of attenuation is reached in the final gain amplifier,
+   *  an abrupt jump to zero gain (infinite dB of attenuation) occurs."
+   * 
+   * fluidsynth is a 24 bit synth, it could (should??) use 144 dB of attenuation.
+   * However the spec makes no distinction between 16 or 24 bit synths, so use
+   * 96 dB here.
+   * 
+   * This magic number was obtained by
+   * powl(10.0L, -96.0L / 20.0L);
+   */
+  static const fluid_real_t MIN_GAIN = 0.000015848931924611135L;
```

Comment:
**Just a remark related to this PR about the the hard coded 96 db of attenuation.**

Actually **fluid_rvoice_calc_amp() , fluid_rvoice_noteoff_LOCAL** and **fluid_rvoice_multi_retrigger_attack()** make use **also** of hard coded **96 db** value:

https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/rvoice/fluid_rvoice.c#L54

https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/rvoice/fluid_rvoice.c#L510

https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/rvoice/fluid_rvoice.c#L572


These hard coded `960.0` should be replaced by a constant (e.g PEAK_ATTENUATION_RANGE = 960.0). 
`static const fluid_real_t PEAK_ATTENUATION_RANGE = 960.0;`

Also we notice that `MIN_GAIN` defined is this PR should be related to `PEAK_ATTENUATION_RANGE` as this:
`MIN_GAIN = powl(10.0L, -PEAK_ATTENUATION_RANGE / 200.0L)`

Probably, the above remark is applicable to the amount of default modulator here in fluid_synth.c  ?:
https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/synth/fluid_synth.c#L308
https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/synth/fluid_synth.c#L320
https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/synth/fluid_synth.c#L384
https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/synth/fluid_synth.c#L412
https://github.com/FluidSynth/fluidsynth/blob/b844dd1f04870f288b7a7d8499ff4a1e08d9539f/src/synth/fluid_synth.c#L469

Reply:
Yes I aggree we could introduce this `PEAK_ATTENUATION_RANGE` constant. However this should be part of another PR. Feel free to do so.

I also wanted to write `MIN_GAIN = pow(10.0, -960.0 / 200.0)`, however as `MIN_GAIN` being `static const` it requires a constant expression to init at compile time. Thus I just hardcoded the value because I didnt want to init some static var in fluid_conv.c and pull them over at runtime.